client/informant/src/lib.rs

```diff
@@ -97,7 +101,18 @@ pub fn build(service: &impl AbstractService, format: OutputFormat) -> impl futur
 			last_best = Some((n.header.number().clone(), n.hash.clone()));
 		}
 
-		info!(target: "substrate", "✨ Imported #{} ({})", Colour::White.bold().paint(format!("{}", n.header.number())), n.hash);
+		match &format {
+			OutputFormat::Coloured { prefix } => info!(
+				target: "substrate",
+				"✨ {}Imported #{} ({})",
+				prefix, Colour::White.bold().paint(n.header.number().to_string()), n.hash,
+			),
+			OutputFormat::Plain { prefix } => info!(
+				target: "substrate", "✨ {}Imported #{} ({})",
+				prefix, n.header.number(), n.hash,
+			),
+		}
```

Comment:
Note: I changed the format for plain, I removed entirely the color

Reply:
I think there is already something that strips the colors. Maybe it's not worth keeping the arms like that (here and in the display.rs) and just display the things with colors. I don't want to go to far in a refactoring... but that's reasonable